code/modules/reagents/machinery/grinder.dm

```diff
@@ -372,7 +372,7 @@
 		if(!container.standard_pour_into(user, src))
 			. = ..()
 	else
```

Comment:
I don't see where it checks for if object has reagents at all, it just prevents putting ANY object in as far as I can tell.
Which will break the grinding part of grinder.
Add `else if (LAZYLEN(O.reagents) . = ..()` before this branch maybe?

Reply:
^